### PR TITLE
Installation notes

### PR DESCRIPTION
**Pull request by nimasmi**
- Update README with things I learnt while adding this to another project
- Add Pygments dependency

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
     install_requires=[
         'Markdown==2.6.2',
         'bleach==1.4.2',
+        'Pygments==2.1.3',
```
Comment (zerolab) [inline setup.py]: if "Syntax highlighting using codehilite is an optional feature", shouldn't Pygments be an optional requirement too?

Comment (nimasmi) [inline setup.py]: Having it there doesn't do a lot, until you add the CSS file. I'm ambivalent about whether this stays here and we amend the wording, or whether we remove the requirement, and insert instructions on adding it manually.

**Comment by zerolab** [comment]
@tm-kn this one is ready to go as well ;)

**Comment by tm-kn** [comment]
@zerolab it had the wrong base branch set. I'll take a look at changing README at some point and may rebase this one.

**Comment by danihodovic** [comment]
Can we re-open this? It's not at all obvious on how to get code highlighting to work right now. The docs in this PR helped me to set it up.

**Comment by zerolab** [comment]
opened a new one - #62. Should hopefully not be long to get that in

**Pull request closed without merging**